theseus_gui/src/pages/Browse.vue

```diff
@@ -66,16 +91,26 @@ const switchPage = async (page) => {
   else searchStore.offset = (searchStore.currentPage - 1) * searchStore.limit
   await getSearchResults()
 }
-
-const handleReset = async () => {
-  searchStore.resetFilters()
-  await getSearchResults()
-}
 </script>
 
 <template>
   <div class="search-container">
     <aside class="filter-panel">
+      <Card
+        v-if="instance"
+        class="instance-card button-base"
+        @click="$router.push(`/instance/${encodeURIComponent(instance.path)}`)"
+      >
+        <Avatar
+          :src="convertFileSrc(instance.metadata.icon)"
+          :alt="instance.metadata.name"
+          size="sm"
+        />
+        <div class="instance-card__info">
+          <span class="title">{{ instance.metadata.name }}</span>
+          {{ instance.metadata.game_version }} {{ instance.metadata.loader }}
+        </div>
+      </Card>
```

Comment:
This should also be on the project page IMO, it should also have some text saying "selected" instance".

It feels a little cramped now in search too. To fix this, I would make sure the sidebar width is the same on every page (like knossos)